CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+# Contributing
+
+Thank you for taking the time to look at this document, which provides some 
```

Comment:
Maybe we can do something like, "Thanks for being interested in contributing to the `rustwasm/book` repo! If you want to get involved in any way or just want to file an issue please look below" this way it's a little more "informal" and exciting! We're super stoked to have people contribute!